lib/removePipelineExtras.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+var Cesium = require('cesium');
+var defined = Cesium.defined;
+
+module.exports = removePipelineExtras;
+
+function removePipelineExtras(gltf) {

```

Comment:
Actually, I didn't realize that the recursive walk was for removing objects - I see what you mean in that case. When I'm adding the extras objects though, I would need some kind of separate reference for which objects are glTF objects, right?